src/Components/MainPage/MainPage.js

```diff
@@ -28,6 +33,9 @@ const MainPage = () => {
     fetchData();
   }, []);
 
+  /*
+  Runs on load and if current token changes
+   */
```

Comment:
The comment is not really telling me what it does just when it runs. Not super useful